### PR TITLE
Io tracker3

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/ioTracker/ioTracker3/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    "light": {
+      "title": "Light",
+      "unit": "lux",
+      "type": "number",
+      "description": "The light in lux.",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "maxAccelerationNew": {
```
Comment (mariusgiger) [inline types/ioTracker/ioTracker3/default.schema.json:44]: why new?

File: types/ioTracker/ioTracker3/gps.schema.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    },
+    "latitude": {
+      "title": "Latitude",
+      "unit": "°",
+      "type": "number",
+      "description": "Latitude",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "altRef": {
```
Comment (mariusgiger) [inline types/ioTracker/ioTracker3/gps.schema.json:29]: fully qualify the dataKey

Comment (mariusgiger) [reply types/ioTracker/ioTracker3/gps.schema.json:29]: for non standard chargon, prevent using abbreviations

File: types/ioTracker/ioTracker3/default.schema.json [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      "enum": ["BUTTON", "MOVEMENT", "GPIO"],
+      "hideFromKpis": true
+    },
+    "crc": {
+      "title": "Cyclic redundancy check",
+      "description": "Cyclic redundancy check",
+      "type": "integer",
+      "hideFromKpis": true
+    },
+    "batteryLevel": {
```
Comment (mariusgiger) [inline types/ioTracker/ioTracker3/default.schema.json:20]: lifecycle topic?

File: types/ioTracker/ioTracker3/uplink.js [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+  // Redundant data is only used as flags
+  delete decoded.containsGps;
+  delete decoded.containsOnboardSensors;
+  delete decoded.containsSpecial;
+
+  if (decoded.sensorContent !== undefined) {
+    delete decoded.sensorContent;
+  }
+
+  emit("sample", { data: decoded, topic: "default" });
```
Comment (mariusgiger) [inline types/ioTracker/ioTracker3/uplink.js:511]: it is clearer if you build the sample by specifying keys and values instead of just dropping keys

**Review by mariusgiger — COMMENTED** [review]
looks good, a few minor comments

File: types/ioTracker/ioTracker3/default.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/ioTracker/ioTracker3/default.schema.json:44]: Well its the new max acceleration in comparison to the old

Comment (mariusgiger) [reply types/ioTracker/ioTracker3/default.schema.json:23]: so it is the difference?

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**